Gemfile

```diff
@@ -4,6 +4,8 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'aws-actionmailer-ses', git: 'https://github.com/aws/aws-actionmailer-ses-ruby', branch: 'init'
```

Comment:
why branch init here? Is this just temporary for testing until we merge the other PR?

Reply:
Yeah this was temporary. I'm going to again keep this here until we release aws-sdk-rails and all four gems have been published.